### PR TITLE
Header Development -TopBar

### PR DESCRIPTION
**Pull request by Rowot**

### CODE REVIEW

File: src/components/Header/header.module.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.container {
+  align-items: center;
+  background-color: #f0f0f0;
```
Comment (javascriptbrew) [inline src/components/Header/header.module.scss]: define color in the constant.scss and use it here

File: src/components/Header/header.module.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  justify-content: space-between;
+  margin: 1rem;
+  padding: 1rem;
+
+  .restro__details {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    .restro__title {
+      color: #000;
```
Comment (javascriptbrew) [inline src/components/Header/header.module.scss]: define color in the constant.scss and use it here

File: src/components/Header/header.module.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      font-family: Open Sans;
+      font-size: 1.25rem;
+      font-style: normal;
+      font-weight: 700;
+      line-height: normal;
+      text-align: center;
+    }
+    .table__num {
+      color: #000;
+      font-family: Open Sans;
```
Comment (javascriptbrew) [inline src/components/Header/header.module.scss]: define font family in the constant.scss and use it here

File: src/components/Header/header.module.scss (hunk 1 above)
Comment (Rowot) [reply src/components/Header/header.module.scss]: Addressed

File: src/components/Header/header.module.scss (hunk 2 above)
Comment (Rowot) [reply src/components/Header/header.module.scss]: Addressed

File: src/components/Header/header.module.scss (hunk 3 above)
Comment (Rowot) [reply src/components/Header/header.module.scss]: Addressed

**Pull request merged**